### PR TITLE
Remaining cleanup work on Microsoft.Extensions.Logging.Console

### PR DESCRIPTION
**Pull request by maryamariyan**
cc: @eerhardt @tarekgh 

- [x] Adds Microsoft.Extensions.Logging.Console.Tests to the solution file
- [x] Remove and sort usings
- [x] Remove IAnsiSystemConsole and AnsiSystemConsole 
- [x] Flip ifdef to check with NETCOREAPP
- [x] Add missing xml docs on ConsoleFormatterNames

### CODE REVIEW

File: src/libraries/Microsoft.Extensions.Logging.Console/tests/ConsoleLoggerExtensionsTests.cs [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-using System.Text.Json;
-using System.Text.Encodings.Web;
-using System.IO;
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Console;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
+using System;
```
Comment (gfoidl) [inline src/libraries/Microsoft.Extensions.Logging.Console/tests/ConsoleLoggerExtensionsTests.cs:11]: Shouldn't the system-namespaces come first, then the rest?
Cf. [C# Coding Style Pt. 6](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md)

Comment (eerhardt) [reply src/libraries/Microsoft.Extensions.Logging.Console/tests/ConsoleLoggerExtensionsTests.cs:11]: Yes. It is in our `.editorconfig` as well:

https://github.com/dotnet/runtime/blob/84c2974ab6b4205366b50484b21416ab920d9424/.editorconfig#L82

@maryamariyan - what did you use to sort these? Visual Studio or something else? VS should have respected the .editorconfig setting.

File: src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleFormatterNames.cs [hunk 2]
```diff
@@ -3,10 +3,24 @@
...
+    /// </summary>
     public static class ConsoleFormatterNames
     {
+        /// <summary>
+        /// Reserved name for simple console formatter
+        /// </summary>
         public const string Simple = "simple";
+
+        /// <summary>
+        /// Reserved name for  json console formatter
```
Comment (eerhardt) [inline src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleFormatterNames.cs:17]: (nit) double space here and in systemd below.

**Review by eerhardt — APPROVED** [review]
Looks good beyond the 2 minor comments.

File: src/libraries/Microsoft.Extensions.Logging.Console/tests/ConsoleLoggerExtensionsTests.cs (hunk 1 above)
Comment (maryamariyan) [reply src/libraries/Microsoft.Extensions.Logging.Console/tests/ConsoleLoggerExtensionsTests.cs:11]: Used VS but it put Microsoft.Extensions namespaces above System.
It also was wrong in the sense that "Remove and Sort unused usings" removed more than it should have but that I fixed manually.

I'll fix this up.

File: src/libraries/Microsoft.Extensions.Logging.Console/src/AnsiParser.cs [hunk 3]
```diff
@@ -59,11 +59,11 @@ public void Parse(string message)
                 {
                     if (span[i + 3] == 'm')
                     {
-    #if (NETSTANDARD2_0 || NETFRAMEWORK)
-                        if (ushort.TryParse(span.Slice(i + 2, length: 1).ToString(), out ushort escapeCode))
-    #else
+#if NETCOREAPP
                         if (ushort.TryParse(span.Slice(i + 2, length: 1), out ushort escapeCode))
```
Comment (tarekgh) [inline src/libraries/Microsoft.Extensions.Logging.Console/src/AnsiParser.cs:63]: I am still thinking it is worth it to parse manually and avoid the whole #if usage.

**Pull request merged**